virttest/ssh.py

```diff
@@ -0,0 +1,30 @@
+from virttest import error_context
+
+
+@error_context.context_aware
+def get_image_filename(server, image_path, user=None, port=None,
+                       host_key_check=None):
+    """
+    Form remote image filename with uri:
+    ssh://[USER@]SERVER[:PORT]/PATH[?host_key_check=HOST_KEY_CHECK]
+    :param server: The remote ssh server
+    :param image_path: Path to the disk image on remote server
+    :param user: Remote user, ff not specified, then the local
+                 username is tried
+    :param port: Port number on which sshd is listening(The default is 22)
+    :param host_key_check: 'yes': to use the local '.ssh/known_hosts' file,
+                           the default is yes
+                           'no': to turn off known-hosts checking
+                           'fingerprint': to check the host key matches a
+                           specific fingerprint, e.g.
+                           'md5:78:45:8e:14:57:4f:d5:45:83:0a:0e:f3...',
+                           note 'sha1:' can also be used as a prefix, but
+                           OpenSSH only use md5 to print fingerprints
+    :return: remote image filename
+    """
+    uri = "ssh://{user}{host}{path}{key_check}"
```

Comment:
Seems like there is no port element in the URI. 
You provide an argument `port` but didn't use it.
I think you should whether add the `port` into `uri = "ssh://{user}{host}{path}{key_check}"`
like uri = "ssh://{user}{host}{port}{path}{key_check}" according to the doc string.

Reply:
My mistake, updated now, thanks.